CHANGES

```diff
@@ -120,6 +115,123 @@ Tools
   Verbose Compat vernacular, since these warnings can now be silenced or
   turned into errors using "-w".
 
+Bugfixes
+
```

Comment:
I tend to agree with you, that the list is growing too much and not so helpful. Having structured commit messages with a "component" field would make it possible to do automatically what you suggest, and I'll try to make a proposal for the next versions.

Meanwhile, what should I do for 8.6beta1? I can still replace the list with "Many bug fixes", it could make the CHANGES file more readable. What do you think?